### ISSUE TITLE
[PQ] Wayshrine of Sigmar

### ISSUE THREAD

**Issue by Scrilian** [WarEmu/WarBugs#11590]
Destro side Ostland PQ - Stage 3 Oswald Striker hero boss has way too much hp. Even with 10 ppl it was barely in time, boss two-shots level-appropriate and geared tanks.
Maybe reduce hp and damage by at least 30-50%.

**Comment by Lheana** [comment]
Have you tried other PQs with hero? This NPC has nothing special except being a hero.

**Comment by Scrilian** [comment]
No, I have not. But I'm pretty sure that a level appropriate group of 10 ppl should not resort to kiting a hero-boss. Instead they probably should be able to tank-n-spank it, esp. at that level healers don't have group heals and AP gain tactics.

**Comment by Lheana** [comment]
Have you tried again the PQ? IS it still an issue?
